index.html

```diff
@@ -479,10 +481,10 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
         <span class="alias">Alias: <b>collect</b></span>
         <br />
         Produces a new array of values by mapping each value in <b>list</b>
-        through a transformation function (<b>iteratee</b>). The <tt>iteratee</tt>
-        is passed three arguments: the <tt>value</tt>, then the <tt>index</tt>
-        (or <tt>key</tt>) of the iteration, and finally a reference to the entire
-        <tt>list</tt>.
+        through a transformation function (<a href="#iteratee"><b>iteratee</b></a>).
+        The <tt>iteratee</tt> is passed three arguments: the <tt>value</tt>,
```

Comment:
nitpick: could you not make the iteratee param an html `<tt>` element? Other than here (and I know it was in the original), the underscore function params are `<b>` elements at first occurrence, and the callback's params are the `<tt>` elements.